lib/server/services/dpl/DplProcessService.js

```diff
@@ -77,7 +77,13 @@ class DplProcessService {
             raw: true,
             attributes: ['id'],
         })).map(({ id }) => id);
-        return (await DplDetectorRepository.findAll({ where: { id: { [Op.in]: ids } } })).map(dplDetectorAdapter.toEntity);
+
+        const detectors = await DplDetectorRepository.findAll({ where: { id: { [Op.in]: ids } } });
+        const sortedDetectors = detectors
+            .map(dplDetectorAdapter.toEntity)
+            .sort((a, b) => a.name.localeCompare(b.name));
+
+        return sortedDetectors;
```

Comment:
You are on the right tracks. However, this works in this particular case, but it doesn't if we apply pagination.
For example, if you have 3 items, let's say

```js
id: 1, name: 'aaa',
id: 2, name: 'ccc',
id: 3, name: 'bbb',
```
If you get the first two items then sort, you will have `aaa` and `ccc`, but you would expect `aaa`, `bbb`.

To solve this, you should apply ordering directly in the query, using https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#ordering

This of course apply to the other modifications you did.